src/Lachain.Core/RPC/HTTP/Web3/AccountServiceWeb3.cs

```diff
@@ -12,30 +12,39 @@
 using System.Collections.Generic;
 using Lachain.Crypto;
 using Lachain.Utility.Serialization;
+using Lachain.Core.Vault;
+using Lachain.Logger;
+using System.Security.Cryptography;
 
 namespace Lachain.Core.RPC.HTTP.Web3
 {
     public class AccountServiceWeb3 : JsonRpcService
     {
+        private static readonly ILogger<AccountServiceWeb3> Logger =
+            LoggerFactory.GetLoggerForClass<AccountServiceWeb3>();
+
         private readonly IStateManager _stateManager;
         private readonly ISnapshotIndexRepository _snapshotIndexer;
         private readonly IContractRegisterer _contractRegisterer;
         private readonly ISystemContractReader _systemContractReader;
         private readonly ITransactionPool _transactionPool;
+        private readonly IPrivateWallet _privateWallet;
 
-        
-
+        private readonly DefaultCrypto crypto = new DefaultCrypto();
```

Comment:
I think it should be something like this: https://github.com/LATOKEN/lachain/blob/dev/src/Lachain.Core/Blockchain/Operations/TransactionSigner.cs#L12

Reply:
fixed, thanks